zammad/templates/statefulset.yaml

```diff
@@ -74,6 +74,13 @@ spec:
         - name: "{{ $key }}"
           value: "{{ $value }}"
         {{- end }}
+        {{ if or .Values.envConfig.elasticsearch.pass .Values.secrets.elasticsearch.useExisting }}
+        - name: "ELASTICSEARCH_PASSWORD"
```

Comment:
why is this env var added?
imho its not used in the container.

Reply:
it's used in the elasticsearch-init script, when username / passwort is required to talk with elasticsearch during elasticsearch initialisation

Reply:
Which step you're talking about exactly?
The credentials are directly used from the variables: https://github.com/zammad/zammad-helm/blob/master/zammad/templates/configmap-init.yaml#L14-L15